extensions/arc/src/extension.ts

```diff
@@ -38,9 +38,11 @@ export async function activate(context: vscode.ExtensionContext): Promise<arc.IE
 		}
 		const dialog = new ConnectToControllerDialog(treeDataProvider);
 		dialog.showDialog();
-		const model = await dialog.waitForClose();
-		if (model) {
+		try {
+			const model = await dialog.waitForClose();
 			await treeDataProvider.addOrUpdateController(model.controllerModel, model.password);
+		} catch (err) {
+			console.log(`Error connecting to Controller dashboard ${err}`);
```

Comment:
This message isn't accurate - we're not "connecting to a controller dashboard" - but rather the controller itself. The dashboard has nothing to do with this.

Reply:
changed back to old code.